src/DropDown/DropDown.stories.mdx

```diff
@@ -0,0 +1,64 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { DropDown } from './DropDown.tsx';
+
+export const itemsWithImages = [
```

Comment:
Does this need to be exported?

Reply:
No, but, it helps with the highlighting in the editor.